### PR TITLE
[Keyboard] Add a new keyboard 'plum47'

### PR DESCRIPTION
**Pull request by awh-tokyo**
## Description

This adds support for the Plum47, a keyboard kit manufactured and sold domestically in Japan by @takagi.  It's based on [a commit](https://github.com/takagi/qmk_firmware/commit/cd03bf6e1173726505524f39d292f4d0b368853f) made by the manufacturer to an old fork of QMK that was never completed and a pull request never submitted.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/plum47/config.h [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// Copyright 2022 Andrew Hamilton (@awh-tokyo)
```
Comment (zvecr) [inline keyboards/plum47/config.h:1]: This file should be removed

File: keyboards/plum47/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+ * |      |      |      |      |      |             |      | Next | Vol- | Vol+ | Play |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_RAISE] = LAYOUT(
+    KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_BSPC,
+    KC_DEL,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_MINS, KC_EQL,  KC_LBRC, KC_RBRC, KC_BSLS,
+    _______, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______, _______, KC_PGUP, KC_PGDN, _______,
+    _______, _______, _______, _______, _______,     _______,      _______, KC_MNXT, KC_VOLD, KC_VOLU, KC_MPLY
+)
+
```
Comment (zvecr) [inline keyboards/plum47/keymaps/via/keymap.c:82]: Keymap needs a 4th layer

File: keyboards/plum47/rules.mk [hunk 3]
```diff
@@ -0,0 +1 @@
+VIA_ENABLE = yes
```
Comment (zvecr) [inline keyboards/plum47/rules.mk:1]: This should enabled in a `via` keymap
```suggestion
# This file intentionally left blank
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/plum47/rules.mk (hunk 3 above)
Comment (zvecr) [reply keyboards/plum47/rules.mk]: This file needs to be put back with the suggested comment.

**Review by zvecr — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**